pkg/network/ebpf/c/protocols/protocol-classification-defs.h

```diff
@@ -15,6 +15,19 @@
 // The minimal HTTP request has 16 characters: GET x HTTP/1.1\r\n
 #define HTTP_MIN_SIZE 16
 
+// Every kafka message encodes starts with:
+//  * 4 bytes for the size of the payload
+//  * 2 bytes for api key
+//  * 2 bytes for api version
+//  * 4 bytes for correlation id
+// Reference: https://kafka.apache.org/protocol.html#protocol_messages
+#define KAFKA_MIN_SIZE 12
+
+// Max today is 13 for fetch (https://kafka.apache.org/protocol.html#protocol_messages)
+#define KAFKA_MAX_VERSION 13
+
+#define KAFKA_MAX_API 67
```

Comment:
Consider extracting the Kafka related consts to different files, so the Kafka parsing can use it